frontend/public/components/masthead-toolbar.jsx

```diff
@@ -93,6 +93,7 @@ class MastheadToolbarContents_ extends React.Component {
     if (window.SERVER_FLAGS.inactivityTimeout >= 300) {
       window.addEventListener('click', _.throttle(this._onInactivityTimeout, 500));
       window.addEventListener('keydown', _.throttle(this._onInactivityTimeout, 500));
+      this._onInactivityTimeout();
```

Comment:
Do you know if the `OPENSHIFT` flag and `user` prop is resolved as this point? We could be racing.

Not new, but this method would be better called `_resetInactivityTimeout`. `_onInactivityTimeout` sounds like you were inactive for too long.